db/seeds.rb

```diff
@@ -53,3 +61,21 @@
 bach.pieces.create(opus: 825, number: 1, type_of_piece: "Prelude-Fugue", composition_date: 1722, multiple_instruments: false, main_instrument: "piano", key_signature: "C-sharp major", nickname: 'Well Tempered Klavier')
 
 bach.pieces.create(opus: 816, number: 5, type_of_piece: "Suite", composition_date: 1722, multiple_instruments: false, main_instrument: "piano", key_signature: "French")
+
+steinway.pianos.create!(model: "Model D", style: "Concert Grand", price: 150000, color: "black", acoustic: true) #length: "8ft 11in")
```

Comment:
Are you planning on adding length as an attribute? If so - I would recommended just doing it in inches and not bothering with feet.